docker/bookkeeper/Dockerfile

```diff
@@ -17,9 +17,9 @@
 # - github.com/apache/bookkeeper/blob/branch-4.9/docker/Dockerfile
 #
 
-FROM apache/bookkeeper:4.9.2
+FROM apache/bookkeeper:4.11.1
```

Comment:
This base image downloads `bookkeeper-server-4.11.1-bin.tar.gz` (as per this [layer](https://hub.docker.com/layers/apache/bookkeeper/4.11.1/images/sha256-87f13f6d73d44a01334e3b2bc1fc4796149677fc52c17097f9ccf82358c8ed92?context=explore)) and then we bring in `bookkeeper-all-${BK_VERSION}-bin.tar.gz` and use this new file. As a result, the image has an extra `bookkeeper-server-4.11.1-bin.tar.gz` file, adding to the image size.  This is a preexisting issue, which is present in our older images too. Would it make sense to add a command to delete that redundant binary here?

Reply:
@ravisharda I cannot find any extra `bookkeeper-server-4.11.1-bin.tar.gz` inside the container (I'm looking into it and cannot find it). I think that the reason is that this file is already being deleted in the Bookkeeper Dockerfile: https://github.com/apache/bookkeeper/blob/8d3060e925e0fb06c7a1880155a809fe28ea052b/docker/Dockerfile#L48
So, I think that what you are suggesting is already done, right?